src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyExecutionManager.cs

```diff
@@ -23,6 +23,8 @@ internal class ParallelProxyExecutionManager : ParallelOperationManager<IProxyEx
     {
         #region TestRunSpecificData
 
+        private bool abortRequested = false;
```

Comment:
This code requires comments on how the `explicit` and `implicit` aborts differ.

Reply:
Fixed!!